src/service/video-manager-impl.js

```diff
@@ -493,6 +544,47 @@ class VideoEntry {
   }
 
   /**
+   * Called when the orientation of the device changes
+   * @param {boolean} isLandscape
+   * @private
+   */
+  orientationChanged_(isLandscape) {
+    // Put the video in/out of fullscreen depending on screen orientation
```

Comment:
return early if `this.loaded_` is `false`. What can happen is orientation change occurring before the `layoutCallback` of a video player is called in which case their `iframe` is null. Now at the moment we can't get to that case because of the `this.getPlayingState() == PlayingStates.PLAYING_MANUAL` check but I like to keep the code safe since later supporting  AUTOPLAY can surface the race condition.